doc/modules/calibration.rst

```diff
@@ -85,18 +90,16 @@ predict values larger than 0 for this case, thus moving the average
 prediction of the bagged ensemble away from 0. We observe this effect most
 strongly with random forests because the base-level trees trained with
 random forests have relatively high variance due to feature subsetting." As
-a result, the calibration curve also referred to as the reliability diagram
-(Wilks 1995 [2]_) shows a characteristic sigmoid shape, indicating that the
-classifier could trust its "intuition" more and return probabilities closer
+a result, the calibration curve shows a characteristic sigmoid shape, indicating that
```

Comment:
Are you against the idea of mentioning that what we call calibration plots are also known as reliability diagrams (I am not against moving such a statement elsewhere)?

Reply:
Not at all, but it is now at the top and not awkwardly in the comments for random forests.